src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -273,6 +273,16 @@ def _normalize_properties(self, user, subscriptions, is_service_principal, cert_
                 if hasattr(s, 'home_tenant_id'):
                     subscription_dict[_HOME_TENANT_ID] = s.home_tenant_id
                 if hasattr(s, 'managed_by_tenants'):
+                    if s.managed_by_tenants is None:
+                        # managedByTenants is missing from the response. This is a known service issue:
+                        # https://github.com/Azure/azure-rest-api-specs/issues/9567
+                        # pylint: disable=line-too-long
+                        from azure.cli.core.azclierror import UserFault
+                        raise UserFault("Invalid profile is used for cloud '{cloud_name}'. "
```

Comment:
Can we define an error sub-class may be InvalidProfileError(UserFault) and raise the specific error? UserFault is designed as an abstract class and is not recommended to be raised directly.

Reply:
This is only a temporary mitigation to the service bug, it should actually be an `Unsupported API version` error.

Reply:
@houk-ms , you can use `ABC` and `abstractmethod` (`from abc import ABC, abstractmethod`) to define your class and make the parent class can't be instantiated.

Reply:
@arrownj sure, we can do that.

Reply:
@jiasli In this case, cosidering use `from azure.cli.core.azclierror import UnknownError` to replace UserFault if you don't define a new error type

Reply:
Let's use `CLIError` for now until the error can be classified as `UnsupportedAPIError` after the service is fixed.